bindings/swift/README.md

```diff
@@ -0,0 +1,64 @@
+# OpenDAL Swift Binding
+
+## Using the Package
+
+### Build C Dependencies
+
+The Swift binding depends on the C binding to OpenDAL. Before using this package, you need to build the C library first:
+
+```
+cd bindings/swift
+make build-c
+``` 
+
+To check whether the package is ready to use, simply run the test:
+
+```
+make test
+```
+
+### Add Dependency to Your Project
+
+The package manifest is not located at the root directory of its repository. To use it, add the path of this package to the `Package.swift` manifest of your project:
+
+```swift
+// swift-tools-version:5.7
+import PackageDescription
+
+let package = Package(
+  name: "MyTool",
+  dependencies: [
+    .package(path: "/path/to/opendal/bindings/swift/OpenDAL"),
```

Comment:
Just for question (we don't need to do this in this PR).

How will we release a swift package? Is there a `crates.io` for `swift`?

Reply:
There is no official package registry for Swift. Similar to golang, Swift packages can be hosted on GitHub or other distributed platforms.